base/interfaces/src/main/java/org/openscience/cdk/interfaces/IMolecularFormula.java

```diff
@@ -249,7 +249,7 @@
      * @param <T>         generic type (of provided class)
      * @return the value stored for the specified description.
      * @see #getProperty(Object)
-     * @see #addProperties(java.util.Map)
+     * @see #getProperties()
```

Comment:
oh, that's actually the wrong "see"... this was following the "java beans" docs pattern: the getter method has a see to the setter method and visa versa. (not a big issue)